app/react/docker/containers/CreateView/CommandsTab/toRequest.ts

```diff
@@ -34,6 +34,11 @@ export function toRequest(
     delete config.Entrypoint;
   }
 
+  // don't include LogConfig object if "Default logging driver" (type === '') is selected
+  if (values.logConfig.type === '') {
```

Comment:
why don't we just return null from `getLogConfig`?

Reply:
For 2 reasons:
1. idk how docker API will handle `HostConfig.LogConfig = null`
2. I could also do
```ts
  HostConfig: {
      ...oldConfig.HostConfig,
      ...( values.lolConfig.type === '' ? {} : {LogConfig: getLogConfig(values.logConfig)),
    },
```
but `oldConfig.HostConfig` may already contain `LogConfig` with `Type === ''` so this solution seemed the best to ensure we don't send a weird payload to the docker API

Reply:
null is translated to nil, which is exactly what you'll have when providing an empty object or `delete HostConfig.LogConfig`

I just preferred to have `getLogConfig` manage its logic. we might forget to do this line if we change things in the future